src/bin/clib-search.c

```diff
@@ -151,6 +114,9 @@ main(int argc, char *argv[]) {
 
   debug_init(&debugger, "clib-search");
 
+  // 15 days
+  clib_cache_init(15 * 24 *60 * 60);
```

Comment:
Not sure what's happening here, but 15 days seems pretty arbitrary. I'm thinking we should make this configurable.